.github/workflows/pull_request_verify.yaml

```diff
@@ -0,0 +1,191 @@
+name: Pull Requests
+
+# When someone comment "/deploy" on anyone PR,
+# this workflow will be triggered and run.
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened, labeled ]
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: full
+  CKB_CLI_VERSION: v0.40.0  
+jobs:
+#job for checkout the ckb
+  checkout: 
+    name: ckb-checkout  
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+        -  Linux-runner
+        -  mac-runner
+    steps:
+    - name: Set env
+      run: echo "CARGO_TARGET_DIR=$(echo $HOME/target)" >> $GITHUB_ENV
+    - uses: actions/checkout@v2
+      with:
+        submodules: false
+        fetch-depth: 2
+        ref: ${{ github.event.pull_request.head.sha }}
+        repository: ${{github.event.pull_request.head.repo.full_name}}
```

Comment:
I think it's OK that uses the default inputs.

Reply:
removed it, uses the default inputs.